Resources/Prototypes/_Lavaland/Entities/Clothing/explorer_suit.yml

```
@@ -55,3 +58,117 @@
+  suffix: Reinforced
+  name: reinforced explorer hood
+  description: An armoured hood for exploring harsh environments. Fully reinforced with goliath plates.
+  components:
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.75
+        Slash: 0.75
+        Piercing: 0.90
+        Heat: 0.70
+        Caustic: 0.90
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)